Gemfile

```diff
@@ -16,8 +16,6 @@ gem "react-rails"
 
 gem 'accountingjs-rails'
 gem "jquery-rails"
-gem "turbolinks"
```

Comment:
It was making it so that I couldn't click on basic links on the breadcrumbs and was driving me nuts.  Joseph said he's been having some of those types of "browser" issues too...  So I ripped it out.